typescript/infra/scripts/print-token-prices.ts

```diff
@@ -50,15 +58,26 @@ async function main() {
 
   const prices = objMap(ids, (_, id) => {
```

Comment:
avoid exiting completely if a coin is not on gecko or has no USD price - common on longer tail or testnets

Reply:
curious if we'd ever run into a mainnet chain where we can't get the price? i guess so

Reply:
ya we have a couple atm e.g. https://www.coingecko.com/en/coins/real-ether